.github/workflows/dotnet.yml

```diff
@@ -35,8 +42,13 @@ jobs:
       run: dotnet paket restore
     - name: Build and Test
       run: dotnet fake run build.fsx
-    - name: Publish NuGet
-      if: matrix.os == 'windows-latest' && github.ref == 'refs/heads/master'
+    - name: Publish prerelease NuGet to GitHub
+      if: matrix.os == 'windows-latest' && github.ref == 'refs/heads/master' && (github.event.inputs.release == null || github.event.inputs.release == 'false')
```

Comment:
Can it be simplified as `github.event.inputs.release != 'true'` ?

Reply:
Yes, it can! Thanks